src/view.help.js

```diff
@@ -4,10 +4,10 @@ class HelpPopup {
     this.store = store;
```

Comment:
I think we should remove local `store` this because `extStore` is global. Keep code a bit saner.

Reply:
It's just the name you know? Or do you mean using extStore in name only?

Reply:
I mean extStore now is global, we should use it everywhere. We shouldn't pass `store` object to each view like this any more.